packages/open-api-gateway/test-integration/open-api-gateway-ts-project.test.ts

```diff
@@ -0,0 +1,105 @@
+import { execSync, ExecSyncOptions } from "child_process";
+import {executeInTempFolderSync} from "./utils/fs-utils";
+import * as fs from "fs";
+import {publishToLocalRegistry} from "./utils/publish-to-local-registry";
+import {startLocalRegistry} from "./utils/start-local-registry";
+import * as path from "path";
+
+describe('OpenAPI Gateway Ts Project Integration Tests', () => {
+
+  beforeAll(async () => {
```

Comment:
consider moving the beforeAll/AfterAll to global setup/teardown as per: https://github.com/aws/aws-prototyping-sdk/blob/73bef22823f8f5851d416c0309e1a6648b57c7df/.projenrc.ts#L343

This is mainly to prevent issues if someone creates another integration test, it will cause failures as multiple verdaccio instances will be created on the same port.

Reply:
Sure - just to clarify that should be global setup within the new open-api-gateway package?

Reply:
Discussed off line and will leave as-is for now :)